components/logins/README.md

```diff
@@ -3,72 +3,134 @@
+but on the understanding that it may be a lengthy discussion.
+
+## Using the Logins component
+
+### Prerequisites
+
+To use this component for local storage of logins data, you will need to know how to integrate appservices components
+into an application on your target platform:
+* **Android**: integrate via the
+  [sync-logins](https://github.com/mozilla-mobile/android-components/blob/master/components/service/sync-logins/README.md)
+  component from android-components.
+* **iOS**: start with the [guide to consuming rust components on
+  iOS](https://github.com/mozilla/application-services/blob/master/docs/howtos/consuming-rust-components-on-ios.md).
+* **Other Platforms**: we don't know yet; please reach out on slack to discuss!
+
+To sync logins data between devices, you will additionally need to integrate the
+[FxAClient component](../fxa-client/README.md) in order to obtain the necessary user credentials and encryption keys,
+and the [SyncManager component](../sync_manager/README.md) in order to orchestrate the syncing process.
+
+### Core Concepts
+
+* A **login record** contains a single saved password along with other metadata about where it should be used.
+Each record is uniquely identified by an opaque string id, and contains fields such as username, password and origin.
+You can read about the fields on a login record in the code [here](./src/login.rs).
+* A **logins store** is a syncable encrypted database containing login records. In order to use the logins store,
+the application must first *unlock* it by providing a secret key (preferably obtained from an OS-level keystore
+mechanism). It can then create, read, update and delete login records from the database.
+  * If the application is connected to Firefox Sync, it can instruct the store to sync itself with the user's
+    server-side logins data. This will upload any local modifications as well as download any new logins data
+    from the server, automatically reconciling records in the case of conflict.
+
+### Examples
+- [Android integration](https://github.com/mozilla-mobile/android-components/blob/master/components/service/sync-logins/README.md)
+
+
+### API Documentation
+- TODO [Expand and update API docs](https://github.com/mozilla/application-services/issues/1747)
+
+
+## Working on the Logins component
+
+### Prerequisites
+
+To effectively work on the Logins component, you will need to be familiar with:
+
+* Our general [guidelines for contributors](../../docs/contributing.md).
+* The [core concepts](#core-concepts) for users of the component, outlined above.
+* The way we [generate ffi bindings](../../docs/howtos/building-a-rust-component.md) and expose them to
+  [Kotlin](../../docs/howtos/exposing-rust-components-to-kotlin.md) and
+  [Swift](../../docs/howtos/exposing-rust-components-to-swift.md).
+* The key ideas behind [how Firefox Sync works](../../docs/synconomicon/) and the [sync15 crate](../sync15/README.md).
```

Comment:
The sync15 README link was broken for me as there doesn't appear to be a `README.md` file in the `sync15` directory.

Reply:
Thanks, good catch. I've added a very very small stub README, but we should flesh it out further in followup work.